build/distributions/CentOS/6/x86_64/DVD/isolinux/isolinux.cfg

```diff
@@ -2,12 +2,18 @@ default simp
 prompt 1
 display boot.msg
 label simp
+  kernel vmlinuz
+  append ks=cdrom:/ks/dvd/auto.cfg initrd=initrd.img audit=1 simp_install=auto simp_disk_crypt
```

Comment:
OCD nit:  Can you please put simp_disk_crypt before simp_install=auto to match the boot option order shown in the UEFI menus?